.github/ISSUE_TEMPLATE/site.yml

```
@@ -0,0 +1,27 @@
+name: Site
+description: Request updates to the documentation site that impact its user interface or functionality.
+labels: ['site']
+body:
+  - type: checkboxes
+    id: issue-type
+    attributes:
+      label: Site issue type
+      description: Select the type(s) of issue you are reporting.
+      options:
+        - label: Design enhancements
+          description: 
+          required: false
+        - label: Functionality improvements
+          description: 
+          required: false
+        - label: UX Enhancements
+          description: 
+          required: false
+        - label: Content management
+          description: 
+          required: false
+    id: resolution
+    attributes:
+      label: Suggested resolution
+      description: Do you have thoughts on how this should be addressed?
+      placeholder: Leave blank if this does not apply.
```

```suggestion
- type: textarea
    id: issue-description
    attributes:
      label: Description
      description: Provide a detailed description of the issue, including any context.
      placeholder: 
    validations:
      required: true
- type: textarea
    id: suggested-change
    attributes:
      label: How should this issue be addressed?
      description: Provide a suggested change if available.
      placeholder: Leave blank if this does not apply.
```

Need a required issue description text area. Add the same as the "Fix content" template.